mantle/system/arch.go

```diff
@@ -27,7 +27,8 @@ func RpmArch() string {
 		return "x86_64"
 	case "arm64":
 		return "aarch64"
-	case "ppc64":
+	/* We only support ppc64le in Fedora derivatives today */
+	case "ppc64", "ppc64le":
```

Comment:
Why not just `ppc64le` here since we don't actually support `ppc64`?

Reply:
yeah. i agree we should just support ppc64le only.